init.zsh

```diff
@@ -1,23 +1,178 @@
-# See https://github.com/Tarrasch/antigen-hs
+#!/usr/bin/env zsh
+# See https://github.com/Tarrasch/antigen-hs#readme
+
 ANTIGEN_HS_HOME=${${0:A}:h}
-if [[ -z "$ANTIGEN_HS_OUT" ]]; then
-    ANTIGEN_HS_OUT="$HOME/.antigen-hs"
+
+if [[ -z "$ANTIGEN_HS_OUT" ]] ; then
+  ANTIGEN_HS_OUT="$HOME/.antigen-hs"
 fi
-if [[ -z "$ANTIGEN_HS_MY" ]]; then
-    ANTIGEN_HS_MY="$ANTIGEN_HS_HOME/../MyAntigen.hs"
+
+if [[ -z "$ANTIGEN_HS_MY" ]] ; then
+  ANTIGEN_HS_MY="$ANTIGEN_HS_HOME/../MyAntigen.hs"
 fi
 
-antigen-hs-compile () {
-  runghc -i"$ANTIGEN_HS_HOME/" -- "$ANTIGEN_HS_MY"
+ANTIGEN_HS_SANDBOX_STACK="stack"
+ANTIGEN_HS_SANDBOX_CABAL="cabal"
+ANTIGEN_HS_SANDBOX_EMPTY="empty"
```

Comment:
I'm not sure what's best, but maybe it's just better to inline these constants than introducing 3 named constant variables?